cityofaustin/atd-vz-data
# Migrations for cr3 location triggers

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/13055

## Testing
**URL to test:** 
Local

**Steps to test:**

1. Use the readme in the /atd-vzd/ folder to get your local stack up and running with production data and the latest migrations and metadata applied
2. Use the Hasura CLI instructions in that readme to apply the migrations and metadata in this branch. From /atd-vzd/:
```
hasura migrate apply --envfile .env.local
hasura metadata apply --envfile .env.local
```
3. Follow the test steps 2-4 (the first step of those instructions is to manually make the changes that these migrations do for us) in https://github.com/cityofaustin/atd-vz-data/pull/1270
5. You can also run ./vision-zero hasura-console to open your local Hasura console and make sure that the [Event Trigger](http://localhost:9695/console/events/data/manage) called location_from_latlon is no longer present in Hasura.
6. If you want to test the down migration, run:

`hasura migrate apply --down 1 --envfile .env.local`


---
#### Ship list
